### PR TITLE
code of polymer was removed

### PR DESCRIPTION
**Pull request by miguel010**
code of polymer was removed restaurant.

### CODE REVIEW

File: app/views/restaurants/_form.html.erb [hunk 1]
```diff
@@ -1,65 +1,65 @@
 <%= form_for @restaurant, :html => { :multipart => true } do |f| %>
+  <% if @restaurant.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@restaurant.errors.count, "error") %> prohibited this restaurant from being saved:</h2>
 
-  <%= hidden_fields(f, :name, :description, :slogan, :phone_number, :email, :address, :twitter, :instagram, :facebook, :website, :delivery) %>
```
Comment (dsapandora) [inline app/views/restaurants/_form.html.erb]: this hidden field are from poylmer???

File: app/views/restaurants/_form.html.erb [hunk 2]
```diff
@@ -1,65 +1,65 @@
...
+      <ul>
+      <% @restaurant.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
 
-  <%= image_tag @restaurant.image.url(:thumb) if @restaurant.image? %>
```
Comment (dsapandora) [inline app/views/restaurants/_form.html.erb]: this image is not necessary!!!

File: app/views/restaurants/_form.html.erb [hunk 3]
```diff
@@ -1,65 +1,65 @@
...
-  </core-field>
-  <core-field>
-    <paper-input label="Facebook" type="text" layout vertical id="facebook" value="<%= @restaurant.facebook %>"></paper-input>
-  </core-field>
-  <core-field>
-    <paper-input label="Website" type="text" layout vertical id="website" value="<%= @restaurant.website %>"></paper-input>
-  </core-field>
-  <paper-checkbox label="Delivery" id="delivery" checked="<%= @restaurant.delivery %>"></paper-checkbox>
-
-  <div>
```
Comment (dsapandora) [inline app/views/restaurants/_form.html.erb]: the content of this div is missing, please add it without polymer.

File: app/views/restaurants/index.html.erb [hunk 4]
```diff
@@ -1,30 +1,12 @@
 <div class="row">
   <% @restaurants.each do |restaurant| %>
-    <section class="col-xs-3">
-      <%= card_image_div(restaurant, current_user) %>
+    <section class="col-md-4">
+      <div class="card image" style="background-image: url(<%= restaurant.image.url(:thumb).to_s %>);" onclick="vote(<%= restaurant.id %>, '<%= restaurant.name %>' );">
         <paper-ripple class="recenteringTouch" fit></paper-ripple>
-        <div id="users_<%= restaurant.id %>" class="show-users"></div>
```
Comment (dsapandora) [inline app/views/restaurants/index.html.erb]: It just about eliminate the polymer no everything else. Please review it once again

File: app/views/restaurants/index.html.erb [hunk 5]
```diff
@@ -1,30 +1,12 @@
 <div class="row">
   <% @restaurants.each do |restaurant| %>
-    <section class="col-xs-3">
-      <%= card_image_div(restaurant, current_user) %>
+    <section class="col-md-4">
+      <div class="card image" style="background-image: url(<%= restaurant.image.url(:thumb).to_s %>);" onclick="vote(<%= restaurant.id %>, '<%= restaurant.name %>' );">
         <paper-ripple class="recenteringTouch" fit></paper-ripple>
```
Comment (dsapandora) [inline app/views/restaurants/index.html.erb]: It just about eliminate the polymer no everything else. Please review it once again
Add a line note

File: app/views/restaurants/_form.html.erb [hunk 6]
```diff
@@ -1,65 +1,65 @@
 <%= form_for @restaurant, :html => { :multipart => true } do |f| %>
+  <% if @restaurant.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@restaurant.errors.count, "error") %> prohibited this restaurant from being saved:</h2>
```
Comment (dsapandora) [inline app/views/restaurants/_form.html.erb]: this hidden field are from poylmer???
Add a line note
  <%= hidden_fields(f, :name, :description, :slogan, :phone_number, :email, :address, :twitter, :instagram, :facebook, :website, :delivery) %>

File: app/views/restaurants/show.html.erb [hunk 7]
```diff
@@ -1,45 +1,59 @@
...
+<p>
+  <strong>Name:</strong>
+  <%= @restaurant.name %>
+</p>
+
+<p>
+  <strong>Description:</strong>
+  <%= @restaurant.description %>
+</p>
+
```
Comment (dsapandora) [inline app/views/restaurants/show.html.erb]: Some information an funtcionality are missing,

File: app/views/restaurants/vote.json.jbuilder [hunk 8]
```diff
@@ -1 +1 @@
-json.extract! @restaurant, :id, :name, :description, :slogan, :phone_number, :email, :address, :twitter, :instagram, :facebook, :website, :delivery, :created_at, :updated_at, :users
\ No newline at end of file
+json.extract! @restaurant, :id, :name, :description, :slogan, :phone_number, :email, :address, :twitter, :instagram, :facebook, :website, :delivery, :created_at, :updated_at
```
Comment (dsapandora) [inline app/views/restaurants/vote.json.jbuilder]: User  is missing

File: app/views/restaurants/_form.html.erb [hunk 9]
```diff
@@ -1,65 +1,65 @@
...
+    <%= f.label :website %><br>
+    <%= f.text_field :website %>
+  </div>
+  <div class="field">
+    <%= f.label :delivery %><br>
+    <%= f.check_box :delivery %>
+  </div>
+  <div class="actions">
+    <%= f.submit %>
   </div>
```
Comment (dsapandora) [inline app/views/restaurants/_form.html.erb]: After submit.... some code are missing...
-      <div class="button raised primary">		+    <%= f.label :facebook %><br>
-        <div class="center" fit>		+    <%= f.text_field :facebook %>
-          <% if @restaurant.new_record? %>		+  </div>
-            Create		+  <div class="field">
-          <% else %>		+    <%= f.label :website %><br>
-            Update		+    <%= f.text_field :website %>
-          <% end %>		+  </div>
-        </div>
Add it without polymers.

File: app/views/restaurants/_form.html.erb (hunk 2 above)
Comment (miguel010) [inline app/views/restaurants/_form.html.erb]: add the condition pictures

File: app/views/restaurants/index.html.erb (hunk 5 above)
Comment (miguel010) [inline app/views/restaurants/index.html.erb]: corrected.

File: app/views/restaurants/_form.html.erb (hunk 6 above)
Comment (miguel010) [inline app/views/restaurants/_form.html.erb]: corrected, fields were added using.

File: app/views/restaurants/show.html.erb (hunk 7 above)
Comment (miguel010) [inline app/views/restaurants/show.html.erb]: corrected.

File: app/views/restaurants/vote.json.jbuilder (hunk 8 above)
Comment (miguel010) [inline app/views/restaurants/vote.json.jbuilder]: users were added.

File: app/views/restaurants/_form.html.erb (hunk 9 above)
Comment (miguel010) [inline app/views/restaurants/_form.html.erb]: corrected.

**Pull request merged**